packages/create-invoice-form/src/lib/invoice/form.svelte

```diff
@@ -350,7 +349,9 @@
         id="dueDate"
         type="date"
         min={inputDateFormat(formData.issuedOn)}
-        value={inputDateFormat(formData.dueDate)}
+        value={inputDateFormat(
+          new Date(new Date(formData.issuedOn).getTime() + 24 * 60 * 60 * 1000)
```

Comment:
Sometimes users might forget the due date, so we added a basic value in case they forget, for better UX